src/Module.Server/Rewards/CrpgRewardServer.cs

```diff
@@ -29,6 +35,8 @@ internal class CrpgRewardServer : MissionLogic
     private readonly CrpgRatingPeriodResults _ratingResults;
     private readonly Random _random;
     private readonly PeriodStatsHelper _periodStatsHelper;
+    private readonly Dictionary<CrpgPeer, Agent> _agentsThatGotTeamhitByCrpgPeer;
+    private readonly Dictionary<CrpgPeer, CrpgUserUpkeepCompensation> _compensationTable;

```

Comment:
I don't think this dictionary needs to be a field. It sounds like it should be created and returned by CompensateTeamhitPlayers.